temporal/logging.go

```diff
@@ -169,6 +171,14 @@ func (w *workflowOutboundLoggerInterceptor) GetLogger(ctx workflow.Context) log.
 	return withFields(logger, getFields(ctx))
 }
 
+func (w *workflowOutboundLoggerInterceptor) GetMetricsHandler(ctx workflow.Context) client.MetricsHandler {
```

Comment:
This function is in the `logging` file on the `workflowOutboundLoggerInterceptor` but it's dealing with getting metric tags. This doesn't feel like good separation of concerns - would it make more sense to create another interceptor to handle this?

Reply:
Actually, I have a different suggestion - it seems like most of the logic in `logger.go` is to propagate key/value pairs through Temporal workflows and activities. I would think that we would want the same key/value pairs for log entries and metrics. So I might actually refactor `workflowOutboundLoggerIntercepter` into something like `workflowOutboundKeyValuePairIntercepter` or something. Then I would pull out the logging logic out into its own file.